LayoutTests/accessibility/misspelling-range.html

```diff
@@ -1,89 +1,90 @@
 <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
 <html>
 <head>
-<script src="../resources/js-test-pre.js"></script>
-<script>
-if (window.testRunner)
-   testRunner.dumpAsText();
-</script>
+<script src="../resources/accessibility-helper.js"></script>
+<script src="../resources/js-test.js"></script>
 </head>
 <body>
 
 <div contenteditable=true id="content" role="textbox">
 wrods is misspelled aab lotsi nowadays. euep.
 </div>
 
-<p id="description"></p>
-<div id="console"></div>
-
 <script>
-    description("This tests that misspelling ranges are properly retrieved in the fashion that a spell checker would.");
-
-    if (window.accessibilityController) {
-        var content = document.getElementById("content");
-        content.focus();
+let output = "This tests that misspelling ranges are properly retrieved in the fashion that a spell checker would.\n\n";
+jsTestIsAsync = true;
```

Comment:
nit: `jsTestIsAsync` should probably go inside the `if (window.accessibilityController && window.internals)` condition, in case that fails.